### ISSUE TITLE
Conflict with another plugin display long values instead of "date" fields.

### ISSUE THREAD

**Issue by danielporto** [RTO-Websites/dynamic-conditions#4]
Last update affected custom cards created with "ele-custom-skin" (https://www.eletemplator.com/ele-custom-skin/)
When the dynamic-conditions plugin is active, the date fields shown in custom cards are displayed as long values, not date format.
This is affecting pages/archives even if no dynamic condition is set for the field.
Disabling the dynamic-conditions plugin makes the field be parsed correctly as date instead of long value.

please look at the date fields on the top of the cards shown in the figure below.
Expected (and previous to plugin update):
https://www.dropbox.com/s/m4d2op1dnn956j3/Screenshot%202019-03-19%2011.49.48.png?dl=0

Shown (broken):
https://www.dropbox.com/s/vi8j8hu59ktpouf/Screenshot%202019-03-19%2011.50.28.png?dl=0

**Comment by crazypsycho** [comment]
Hello,

thanks for reporting this issue. We have reproduced it and think we can provide a fix very soon.

**Comment by crazypsycho** [comment]
Hello again,

we have fixed this issue now and so it should work now.  Just update to version 1.2.1

**Comment by danielporto** [comment]
Sorry to say that the fix did not work.
I just updated and date values are still shown as long. :-(

**Comment by crazypsycho** [comment]
That´s strange. I have created a loop template with some text-editor-widgets and set dynamic tag to post-date.  Than i added a post-widget and set my template as custom skin. Date is shown as aspected.
Did you use other widgets?
Or did you have a cache plugin?

**Comment by danielporto** [comment]
The field date I use is a custom field created with ACF. Maybe it matters..
No cache. I even downloaded and tried with a localhost deployment. Same
result.
Meanwhile I will check the logs of the server to check if something calls
my attention.
Thanks

**Comment by danielporto** [comment]
I unfortunately haven't found anything that could point to the error in my logs.
Have you guys tried with ACF date fields? 
thanks

**Comment by crazypsycho** [comment]
Hello,

we have tried a lot. Currently it seems that date-time-picker-fields does not work with elementor. 
But we can´t reproduce the issue with date-fields or normal text-fields. 

Can you give us a list of your plugins? So we can look if its a conflict with an other plugin.

**Comment by crazypsycho** [comment]
Hello again,

you can try to delete and reinstall the plugin. Then it should work.

**Comment by danielporto** [comment]
Hello there,
I have reduced the number of plugins to the minimum. I did some tests in my
local deployment so I could try extreme configurations.
https://www.dropbox.com/s/w5macywhbbjdrmz/Screenshot%202019-03-25%2009.47.17.png?dl=0
I have removed all other plugins. The problem persists.
https://www.dropbox.com/s/avb9lmserdd30oe/Screenshot%202019-03-25%2009.49.11.png?dl=0
Surprisingly, the first element is in the correct format but all other
elements of the archive has long values.
I will try reinstalling the plugin now.
Thanks
